packaging/server/build.sh

```diff
@@ -0,0 +1,38 @@
+#! /bin/bash
+# Execute the script line `bash packaging/backend/build.sh ...`
+
+set -e
+
+SCRIPTDIR=$(dirname $(realpath -s "$0"))
+ROOTDIR="$SCRIPTDIR/../.."
+
+CURRENT_DATE=$(date --iso-8601)
+CURRENT_VERSION=$(grep -o '^version = .*$' $ROOTDIR/pyproject.toml | sed 's/version = "\(.*\)"$/\1/' | tr '+' '.')
+CURRENT_COMMIT_SHA=$(git rev-parse --short HEAD)
+
+UNIQ_TAG="$CURRENT_DATE-$CURRENT_VERSION-$CURRENT_COMMIT_SHA"
+PREFIX=ghcr.io/scille/parsec-cloud
```

Comment:
Would be good to add a small comment about this prefix, from what I understand `ghcr.io/scille/parsec-cloud` refers to Github's own docker images repository for our project

Reply:
```suggestion
# We use Github package repository to store our docker's container.
PREFIX=ghcr.io/scille/parsec-cloud
```

WDYT ?

Reply:
👍